style.css

```diff
@@ -149,11 +149,20 @@ header {
 	
 }
 
-.img_me {
+.img_me_box {
 	margin: 0;
 	width: 33.333%;
+	overflow: hidden;
+	
+}
+
+.img_me {
+	width: 100%;
+	background-image: url("./images/sarah-dorweiler-QeVmJxZOv3k-unsplash.jpg");
 	height: 350px;
-	background-color: #FFE2D1;
+	background-position: center; 
+	background-size: cover; 
+
 }
```

Comment:
May or may not affect you, but I've had an issue where media queries won't fire on mobile devices. Fix is to add the following element to the head:
<meta name="viewport" content="width=device-width, initial-scale=1">

Reply:
"<meta name="viewport" content="width=device-width, initial-scale=1">"

Reply:
"meta name="viewport" content="width=device-width, initial-scale=1"
Sorry, github doesn't like stuff between angle brackets, apparently

Reply:
thanks- fixed!